app/scripts/components/common/banner/banner.scss

```
@@ -1,3 +1,12 @@
+@use '$styles/veda-ui-theme-vars.scss' as themeVars;
+
 .usa-banner__button:after {
-    top: 3px;
-}
\ No newline at end of file
+  top: 3px;
+}
+
+.veda_one_padding {
```

What does one padding refer to here? 

```suggestion
.banner--left-aligned {
```